docs/source/guide/zne-3-options.md

```diff
@@ -216,6 +216,72 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+#### Folding gates by layer
+
+Circuit debugging techniques, as proposed in
+*Calderon et al. Quantum (2023)* {cite}`Calderon_2023_Quantum`, utilize the
+ability to apply layerwise folding of a circuit.
+
+A quantum circuit $C$ may be represented as a series of $d$ layers. Each layer $L$
+contains one or more quantum gates acting concurrently on an $n$-qubit
+system:
+
+$$
+C = L_d L_{d-1} \cdots L_2 L_1
+$$.
```

Comment:
Looks like the error is caused by the period _after_ the `$$`. If you move it directly inside it works.

```suggestion
$$C = L_d L_{d-1} \cdots L_2 L_1.$$
```

Reply:
Excellent.  Saved me some headache there. Thank you! :)